src/jit/hwintrinsicxarch.cpp

```diff
@@ -152,6 +152,40 @@ InstructionSet Compiler::isaOfHWIntrinsic(NamedIntrinsic intrinsic)
     return hwIntrinsicInfoArray[intrinsic - NI_HW_INTRINSIC_START - 1].isa;
 }
 
+//------------------------------------------------------------------------
+// isIsSupport: return true if the intrinsic is "get_IsSuppoted"
+//
+// Arguments:
+//    intrinsic -- id of the intrinsic function.
+//
+// Return Value:
+//    true if the intrinsic is "get_IsSuppoted"
+//    Sometimes we need to specially treat "IsSuppoted"
+bool Compiler::isIsSupport(NamedIntrinsic intrinsic)
```

Comment:
Some typos in comments `IsSuppoted` vs `IsSupported`.

Also the method name seems a bit odd. Maybe something more verbose like `isIntrinsicAnIsSupportedPropertyGetter` ?

Reply:
Thanks! Will change.